examples/iDynTreeWrappers/visualizeRobot.m

```diff
@@ -56,53 +65,59 @@
     'neck_roll';
     'neck_yaw'
     };
-%
+
+% Main variable of iDyntreeWrappers used for many things including updating
+% robot position and getting world to frame transforms
 KinDynModel = iDynTreeWrappers.loadReducedModel(jointOrder,'root_link',modelPath,fileName,false);
 
 % create vector of positions
 joints_positions=zeros(KinDynModel.NDOF,1);
 
-% Prepare figure, handles and variables required for the update
-[transforms,linkNames,linkNames_idyn,map,linkMeshInfo,meshHandles,parent,mainHandler]=iDynTreeWrappers.prepareVisualization(KinDynModel,meshFilePrefix);
-% Note: For default visualization these variables can be ignored:
-%       map,linkMeshInfo,meshHandles,parent,mainHandler.
+fakeWorld=[1,0,0,0;0,1,0,0;0,0,1,0.6;0,0,0,1];%eye(4);
```

Comment:
What is `fakeWorld`  ? I suggest to use a more clear name such as `world_H_base`, and remove `%eye(4)` if it is not used.